webapp/store/page/dataExport/DataExportStateType.ts

```diff
@@ -1,12 +1,14 @@
 import { Country } from '@core/country'
 
 export interface DataExportSelection {
-  countryISOs: Array<string>
+  columns: string[]
+  countryISOs: string[]
   variable: string
-  columns: Array<string>
 }
 
 export interface DataExportState {
-  countries: Array<Country>
-  selection: Record<string, DataExportSelection>
+  countries: Country[]
+  selection: {
+    [key: string]: DataExportSelection
+  }
 }
```

Comment:
these changes are not needed. Plus, better to use Record, plus I prefer Array to []